compiler/src/iree/compiler/Codegen/Common/FoldTensorSubsetIntoVectorTransferOps.cpp

```diff
@@ -204,11 +205,99 @@ class FoldInsertSliceIntoTransferWrite final
   }
 };
 
+/// Fold tensor.extract_slice into vector.transfer_write if
+///   1. The tensor.extract_slice op has only one use.
```

Comment:
The pattern looks like it is checking that the `transfer_write` has a single use (the source for the `extract_slice`).
```suggestion
///   1. The vector.transfer_write op has only one use.
```

Reply:
good catch!